test-results/.last-run.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think this folder `test-results/*` should be included in the .gitignore and this json should be removed from the PR.